feature-libs/product-configurator/rulebased/components/attribute/footer/configurator-attribute-footer.component.ts

```diff
@@ -39,16 +39,19 @@ export class ConfiguratorAttributeFooterComponent implements OnInit {
   /**
    * Checks if attribute is a user input typed attribute with empty value.
    * Method will return false for domain based attributes
-   * @param input
+   * @param {string} input - user input
    */
   isUserInputEmpty(input: string): boolean {
     return input !== undefined && (!input.trim() || 0 === input.length);
   }
 
   protected needsUserInputMessage(): boolean {
+    const uiType = this.attribute.uiType;
     return (
       this.attribute.required &&
       this.attribute.incomplete &&
+      (uiType === Configurator.UiType.STRING ||
```

Comment:
Should we add unit test for that new logic?

Reply:
Pick

Reply:
Done directly in epic